coalib/bearlib/languages/documentation/DocstyleDefinition.py

```diff
@@ -21,7 +21,7 @@ class DocstyleDefinition:
 
     @enforce_signature
     def __init__(self, language: str, docstyle: str, markers: (Iterable, str),
-                 metadata: Metadata):
+                 metadata: Metadata, padding: (Iterable, str)):
```

Comment:
Is it possible to provide a default value for this new arg, thereby avoiding breaking the API?